tests/e2e/e2e_test.go

```diff
@@ -101,6 +118,16 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 		ginkgo.DeferCleanup(func() {
 			tests.Outf("Shutting down network\n")
 			require.NoError(network.Stop())
+			if archiveNetworkDirOnTeardown {
+				archivePath, err := filepath.Abs("../../testnet.tar.xz")
+				require.NoError(err)
+				tests.Outf("Archiving network dir %s to %s\n", network.Dir, archivePath)
+				cmd := exec.Command(
+					// Specifying -C ensures that the archive will exclude the path containing the network dir.
+					"tar", "cjf", archivePath, "-C", filepath.Dir(network.Dir), filepath.Base(network.Dir),
+				)
```

Comment:
Is there any way to do this without an `exec`?

Reply:
I guess I'm wondering if this would make more sense as a bash script rather than part of the actual test.

Reply:
Also sent in DM but I'd really prefer the taring to be handled by the script if possible (rather than inside the binary)